packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -64,6 +65,10 @@ final RegExp androidXFailureRegex = RegExp(r'(AAPT|androidx|android\.support)');
 
 final RegExp androidXPluginWarningRegex = RegExp(r'\*{57}'
   r"|WARNING: This version of (\w+) will break your Android build if it or its dependencies aren't compatible with AndroidX."
+  r'|To fix the issue, run:'
+  r'`export ENABLE_FLUTTER_BUILD_PLUGINS_AS_AAR=true` on Linux or MacOS.'
```

Comment:
Here and elswhere: can we avoid using environment variables to control this, and perhaps instead use either pubspec.yaml or settings.gradle/build.gradle?

Reply:
I think environment variables is what you want here, since it gives you the necessary flexibility.  The goal is to turn this on for everyone and remove the variable -- this is just a risk mitigation strategy.

Reply:
/cc @jonahwilliams  - From what I understand, using environment variables elsewhere in the tool has caused us pain and been difficult to explain to customers/get away from when we eventually want to. 

Is it not possible to have a setting in either settings.gradle or build.gradle to control this?